XCUITests/FxScreenGraph.swift

```diff
@@ -172,6 +180,7 @@ func createScreenGraph(_ app: XCUIApplication, url: String = "https://www.mozill
         scene.tap(table.cells["Logins"], to: LoginsSettings)
         scene.tap(table.cells["ClearPrivateData"], to: ClearPrivateDataSettings)
         scene.tap(table.cells["OpenWith.Setting"], to: OpenWithSettings)
+        scene.tap(table.cells["Show Tour"], to: Intro_Organize)
```

Comment:
I cannot find it either :/ I guess it should be defined here @jhugman? :
https://github.com/mozilla-mobile/firefox-ios/blob/0fe9935790ce5bf606b3a0e35084608734f6cc3e/Client/Frontend/Settings/AppSettingsOptions.swift